experimental/framework/data-objects/src/signaler/signaler.ts

```diff
@@ -193,8 +196,8 @@ class SignalerClass
  * Implementation of ISignaler for declarative API.
  * @privateRemarks
  * `factory` part of the type is included here to satisfy the usage in `@fluid-example/presence-tracker`, which is accessing encapsulated API surfaces from this.
- * If this eventually gets promoted to `@public` and/or part of `fluid-framework`, that part of the type should be left as `@internal` or `@alpha`.
- * @internal
+ * If this eventually gets promoted to `@public` and/or part of `fluid-framework`, an alternate LegacySignaler (`@legacy`) should be created to continue exposing `factory`.
```

Comment:
I think it's more likely that this gets deleted after the DIS work is done, right?  Maybe should start pointing the language in that direction instead.

Reply:
I also considered that. I hope that internal factory part can be removed. But hesitant to point to the thing that doesn't have any representation in repo yet and will be renamed at least once before then. Definitely plan to come back here as we make progress - I kept this "private".